pkg/app/web/src/components/application-form/index.tsx

```diff
@@ -194,8 +203,28 @@ export const ApplicationFormTabs: React.FC<ApplicationFormProps> = (props) => {
           onChange={handleChange}
           aria-label="basic tabs example"
         >
-          <Tab label="Add manually" {...a11yProps(0)} />
-          <Tab label="Add from Git (Alpha)" {...a11yProps(1)} />
+          <Tab
+            classes={{ wrapper: classes.tab, labelIcon: classes.labelIcon }}
```

Comment:
Nice 👍  I can say that we should avoid adding CSS directly (using `classes`) like this to avoid spaghetti CSS. Should use `className` instead 👀 
```typescript
className={classes.tabLabel}
```
and at L148 should be changed too